osg_configure/configure_modules/gratia.py

```diff
@@ -704,7 +704,11 @@ def replace_setting(buf, setting, value, xml_file=True):
 
           returns the string with the option string replaced/added
         """
-        value = str(value).strip('"')
+        value = str(value)
+        if xml_file:
+            value = value.replace('"', '&quot;')
+        else:
+            value = value.replace('"', '\"')
```

Comment:
Would it make sense to use a library function here for quoting?  (in case `"` isn't the only character that needs to be escaped?)